Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -17,25 +17,19 @@
           height="44"
           alt="B.C. Government Logo"
         />
-      </router-link>
+          <b-navbar-brand>
+      <h4 class="nav-link" to="/timeLine">HealthGateway</h4>
```

Comment:
This was not a link before and it goes to the same location as the logo. Is that intentional?

Reply:
yes it was intentional

Reply:
just to be sure, it looks like we have three different links all going to timeline, however there is nowhere else to go?